pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -550,8 +550,10 @@ stdenv.mkDerivation {
     expandResponseParams = "${expand-response-params}/bin/expand-response-params";
     shell = getBin shell + shell.shellPath or "";
     gnugrep_bin = if nativeTools then "" else gnugrep;
+    # stdenv.cc.cc should not be null and we have nothing better
+    cc = if nativeTools then "" else cc;
```

Comment:
```suggestion
    # stdenv.cc.cc should not be null and we have nothing better for now.
    # if the native impure bootstrap is gotten rid of this can become `inherit cc;` again.
    cc = if nativeTools then "" else cc;
```